packages/core/admin/ee/server/validation/review-workflows.js

```diff
@@ -21,7 +21,14 @@ const validateUpdateStageOnEntity = yup
   })
   .required();
 
+const validateWorkflowCreateSchema = yup.object().shape({
+  name: yup.string().max(255).required(),
```

Comment:
The name can also not be a duplicate with any other workflow name.

Reply:
👍 I will set a unique flag in the db schema

Reply:
it has to be unique across all workflow stages? not unique inside the same workflow?

Reply:
Workflows can not have the same name - stages within the workflow neither, but across workflows stage names can be duplicated.

Reply:
ah! got it, missunderstood the question :)

Reply:
added this validation in https://github.com/strapi/strapi/pull/16690